spec/values/calculation/max.hrx

```diff
@@ -0,0 +1,215 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `max()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> error/syntax/no_args/input.scss
+a {b: max()}
+
+<===> error/syntax/no_args/error
+Error: At least one argument must be passed.
+  ,
+1 | a {b: max()}
+  |       ^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/invalid_arg/input.scss
+a {b: max(c)}
+
+<===> error/syntax/invalid_arg/error
+Error: c is not a number.
+  ,
+1 | a {b: max(c)}
+  |       ^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/rest/input.scss
```

Comment:
Should this be an error?

Reply:
No—the rest argument causes this to be parsed as the global `max()` function—but it certainly shouldn't be tested here.